edb/schema/objects.py

```diff
@@ -817,8 +820,8 @@ def get_verbosename(
         dname = self.get_displayname(schema)
         return f"{clsname} '{dname}'"
 
-    def __init__(self, *, _private_init: bool) -> None:
-        pass
+    def __init__(self, *, _private_init: uuid.UUID) -> None:
```

Comment:
I'd call it "private_id"

```suggestion
    def __init__(self, *, _private_id: uuid.UUID) -> None:
```